mne/bem.py

```diff
@@ -332,3 +337,108 @@ def cost_fun(x, points):
     origin = x_opt[:3]
     radius = x_opt[3]
     return radius, origin
+
+
+# ############################################################################
+# Create BEM surfaces
+
+@verbose
+def make_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
```

Comment:
Ok, so the function `make_watershed_bem` will just raise an error if no `subject` is provided? Instead should it keep looking for the `SUBJECTS_DIR` var? In any case, the function must look for the `FREESURFER_HOME` env var.
Currently the script `mne_watershed_bem.py` do not check for env vars, but I don't think I'll have to add this check since the `subject` option is required to be set.